increment.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+
+const config = require('./config')
+
+const issueCommentsJson = `${config.source.repo}/issue-comments.json`;
+const prCommentsJson = `${config.source.repo}/pull-comments.json`;
+
+const issueComments = JSON.parse(fs.readFileSync(issueCommentsJson));
+const prComments = JSON.parse(fs.readFileSync(prCommentsJson));
+
+issueComments.map(issue => {
+  const issueNumber = parseInt(issue.issue_url.split('/').pop())
+  issue.issue_url = issue.issue_url.replace(`/issues/${issueNumber}`, `/issues/${issueNumber + config.increment}`);
```

Comment:
This is half baked, still need to add `config.increment` to the gen script, but just stashing this for now.